tmail-backend/tmail-third-party/ai-bot/src/main/scala/com/linagora/tmail/jmap/mail/AiBotSuggestReplyRequest.scala

```diff
@@ -0,0 +1,42 @@
+/** ******************************************************************
+ * As a subpart of Twake Mail, this file is edited by Linagora.    *
+ * *
+ * https://twake-mail.com/                                         *
+ * https://linagora.com                                            *
+ * *
+ * This file is subject to The Affero Gnu Public License           *
+ * version 3.                                                      *
+ * *
+ * https://www.gnu.org/licenses/agpl-3.0.en.html                   *
+ * *
+ * This program is distributed in the hope that it will be         *
+ * useful, but WITHOUT ANY WARRANTY; without even the implied      *
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR         *
+ * PURPOSE. See the GNU Affero General Public License for          *
+ * more details.                                                   *
+ * ****************************************************************** */
+package com.linagora.tmail.jmap.mail;
```

Comment:
mail package references `rfc-8621` jmap mail and maybe do not make sense in this context.

Suggestion:

```suggestion
package com.linagora.tmail.jmap;
```